### PR TITLE
Added Shipatron to 'Apps using Shopify API node'

### PR DESCRIPTION
**Pull request by Stumblor**

### CODE REVIEW

**Comment by Stumblor** [comment]
Thanks! :-)

File: README.md [hunk 1]
```diff
@@ -674,3 +675,5 @@ Used in our live products: [MoonMail][moonmail] & [MONEI][monei]
   https://apps.shopify.com/wholesaler-pro-1?ref=microapps
 [wholesaler-pro]: https://apps.shopify.com/wholesaler-pro-2?ref=microapps
 [youtube-traffic]: https://apps.shopify.com/youtube-traffic?ref=microapps
+[shipatron]: https://shipatron.io
+
```
Comment (lpinca) [inline README.md]: Can you please remove this extra new line?

**Comment by Stumblor** [comment]
👍🏻

**Comment by lpinca** [comment]
Thank you.

**Comment by Stumblor** [comment]
And to you! 😁

**Pull request merged**